mdonatello/mdonatello.py

```diff
@@ -10,11 +10,51 @@
 
 
 class MoleculeVisualizer:
-    def __init__(self, ag, show_atom_indices=False):
-        self.ag = ag.convert_to("RDKit")
-        self.ag_noh = Chem.RemoveHs(self.ag)
-        AllChem.Compute2DCoords(self.ag_noh)
+    """A class for small molecule 2D visualization in jupyter notebook
+
+    Parameters:
+    -----------
+    ag : MDAnalysis.core.groups.AtomGroup
+        An AtomGroup object representing the molecules that need to be visualized.
+    show_atom_indices : bool, optional
+        Whether to display atom indices of the molecule. Default is False.
+    width : int, optional
+        The width of the image in pixels. Default is 300.
+    height : int, optional
+        The height of the image in pixels. Default is 300.
+
+    Methods:
```

Comment:
In general, we follow the Stanard Numpy doc style: https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard

We don't normally have a `Methods` section; they will automatically be built and shown if you have a doc string for each method.

Reply:
Ah I see, thanks yuxuan I will adjust it to the Numpy doc style :)

So mainly description, parameters + return ?